bundles/org.openhab.binding.remehaheating/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,141 @@
+		<label>Remeha Boiler</label>
+		<description>Remeha Home heating system boiler</description>
+		<category>heating</category>
+
+		<channels>
+			<channel id="roomTemperature" typeId="roomTemperature"/>
+			<channel id="targetTemperature" typeId="targetTemperature"/>
+			<channel id="dhwTemperature" typeId="dhwTemperature"/>
+			<channel id="dhwTarget" typeId="dhwTarget"/>
+			<channel id="dhwMode" typeId="dhwMode"/>
+			<channel id="dhwStatus" typeId="dhwStatus"/>
+			<!-- <channel id="dhwBoost" typeId="dhwBoost"/> -->
+			<channel id="waterPressure" typeId="waterPressure"/>
+			<channel id="waterPressureOK" typeId="waterPressureOK"/>
+			<channel id="outdoorTemperature" typeId="outdoorTemperature"/>
+			<channel id="status" typeId="status"/>
+		</channels>
+
+		<config-description>
+			<parameter name="email" type="text" required="true">
+				<context>email</context>
+				<label>Email</label>
+				<description>Remeha Home account email</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<context>password</context>
+				<label>Password</label>
+				<description>Remeha Home account password</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="30" max="3600" unit="s">
+				<label>Refresh Interval</label>
+				<description>Interval to refresh data from Remeha API</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="roomTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Room Temperature</label>
+		<description>Current room temperature</description>
+		<category>temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%.1f °C"/>
+	</channel-type>
+
+	<channel-type id="targetTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>Target room temperature</description>
+		<category>temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state min="5" max="30" step="0.5" pattern="%.1f °C"/>
+	</channel-type>
+
+	<channel-type id="dhwTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Hot Water Temperature</label>
+		<description>Current hot water temperature</description>
+		<state readOnly="true" pattern="%.1f °C"/>
+	</channel-type>
+
+	<channel-type id="dhwTarget">
+		<item-type>Number:Temperature</item-type>
+		<label>Hot Water Target</label>
+		<description>Target hot water temperature</description>
+		<state readOnly="true" pattern="%.1f °C"/>
+	</channel-type>
+
+	<channel-type id="waterPressure">
+		<item-type>Number:Pressure</item-type>
+		<label>Water Pressure</label>
+		<description>System water pressure</description>
+		<category>pressure</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Pressure</tag>
+		</tags>
+		<state readOnly="true" pattern="%.1f bar"/>
```

Comment:
When using %unit% in the pattern, you allow the user to decide what unit to use. With a unithint, you could suggest `bar`

Same apply's to the other channels like temperature